tests/test_parse.py

```diff
@@ -2,7 +2,7 @@
 
 import pytest
 
-from biip import ParseError, ParseResult, parse
+from biip import ParseError, ParseResult, parse, parse_hri
```

Comment:
This needs to be undone.

```suggestion
from biip import ParseError, ParseResult, parse
```